client/src/features/visualizer-vivagraph/components/KeyPanel.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { memo } from "react";
+import { BlockState } from "@iota/sdk-wasm-nova/web";
+import StatsPanel from "~features/visualizer-threejs/wrapper/StatsPanel";
+
+import { SEARCH_RESULT_COLOR, THEME_BLOCK_COLORS } from "../definitions/constants";
+import { ThemeMode } from "../definitions/enums";
+import ColorPanel from "./ColorPanel";
+import "./KeyPanel.scss";
+
+export const KeyPanel = ({ network, themeMode }: { network: string; themeMode: ThemeMode }) => {
+    const statuses: {
+        label: string;
+        state: BlockState | "searchResult";
+    }[] = [
+        {
+            label: "Pending",
+            state: "pending",
+        },
+        {
+            label: "Accepted",
+            state: "accepted",
+        },
+        {
+            label: "Confirmed",
+            state: "confirmed",
+        },
+        {
+            label: "Finalized",
+            state: "finalized",
+        },
+        {
+            label: "Dropped",
+            state: "dropped",
+        },
+        {
+            label: "Orphaned",
+            state: "orphaned",
+        },
```

Comment:
Question, do we need these two?

Reply:
I dont think we can detect them honestly but we can leave them in and during polish phase we see what stays and what leaves